safe/definitions/units.py

```diff
@@ -154,6 +154,22 @@
         }
     ],
 }
+unit_hectare = {
+    'key': 'hectare',
+    'name': tr('Hectare'),
+    'base_unit': False,
+    'plural_name': tr('hectares'),
+    'measure': tr('Area'),
+    'abbreviation': 'ha',
+    'description': tr(
+        '<b>Hectare</b> are a metric unit of measure.'),
```

Comment:
The description is almost the same as square metres. May be you can get the description from https://en.wikipedia.org/wiki/Hectare :) And add the citation from it also